src/app/themes/theme.js

```diff
@@ -1,403 +1,413 @@
+            },
+        },
+        MuiTablePagination: {
+            root: {
+                fontSize: '1em',
+                display: 'grid',
+                justifyContent: 'space-evenly',
+            },
+            spacer: {
+                flex: 'none',
+            },
+            toolbar: {
+                backgroundColor: '#ffffff',
+                margin: '16px',
+                boxShadow: '0px 2px 1px -1px rgb(0 0 0 / 20%), 0px 1px 1px 0px rgb(0 0 0 / 14%), 0px 1px 3px 0px rgb(0 0 0 / 12%)',
+                borderRadius: '64px',
+                display: 'flex',
+                justifyContent: 'center',
+                padding: '8px 32px',
+                paddingRight: '16px',
+            },
+            select: {
+                color: '#156dac',
+                fontWeight: '500',
+            },
+            actions: {
+                color: '#156dac',
+            },
+        },
+        MuiTableRow: {
+            root: {
+                '&:hover': {
+                    backgroundColor: 'rgb(245, 249, 253, 0.9)',
+                },
+            },
+        },
+        MuiTableSortLabel: {
+            active: {
+                color: '#000000',
+            },
+            icon: {
+                color: '#000000',
+            },
+        },
+        MuiFormLabel: {
+            asterisk: {
+                fontSize: '2em',
+                verticalAlign: 'text-top',
+            },
+        },
+        MuiTableSortLabel: {
+            active: {
+                color: '#000',
+            },
+            icon: {
+                color: '#000',
+            },
+        },
+        MuiOutlinedInput: {
+            input: {
+                paddingTop: '18.5px',
+                paddingBottom: '14px',
+            },
+            inputMarginDense: {
+                paddingTop: '14.5px',
+                paddingBottom: '10px',
+            },
+            multiline: {
+                paddingTop: '14.5px',
+                paddingBottom: '10px',
+            },
+            inputMultiline: {
+                height: '256px',
+            },
+        },
+        MuiStepLabel: {
+            root: {
+                flexDirection: 'column',
+            },
+            label: {
+                fontWeight: '500',
+                '&$active': {
+                    fontWeight: '600',
+                },
+            },
+            iconContainer: {
+                paddingRight: '0',
+            },
+        },
+        MuiStepIcon: {
+            root: {
+                fontSize: '1.7em',
+                '&$active': {
+                    boxShadow: '1px 0px 4px 4px #156dac50',
+                    borderRadius: '64px',
+                },
+                '&$completed': {
+                    color: '#356635',
+                },
+            },
+            text: {
+                fontSize: '1rem',
+            },
+        },
     },
-  },
 });
-export default theme;
+export default theme;
```

Comment:
Can you figure out how to have your editor stop removing the "end of file last line" thing? There should be a newline at the end of all of our files